src/jit/morph.cpp

```diff
@@ -6950,34 +7056,72 @@ bool Compiler::fgCanFastTailCall(GenTreeCall* callee)
     }
 #endif
 
+    auto reportFastTailCallDecision = [this, callee](const char* msg, size_t callerStackSize, size_t calleeStackSize) {
+#if DEBUG
+        if ((JitConfig.JitReportFastTailCallDecisions() & 1) == 1)
+        {
+            if (callee->gtCallType != CT_INDIRECT)
```

Comment:
Seems like all this output should be preceded by a string like "Fast tailcall decision: " to make it obvious what is being output (and make it easy to search for)